test/e2e/volume_provisioning.go

```diff
@@ -104,46 +105,44 @@ var _ = framework.KubeDescribe("Dynamic provisioning", func() {
 		ns = f.Namespace.Name
 	})
 
-	/*
-		TODO: enable when GKE is updated with the new API
-		framework.KubeDescribe("DynamicProvisioner", func() {
-			It("should create and delete persistent volumes [Slow] [Volume]", func() {
-				framework.SkipUnlessProviderIs("openstack", "gce", "aws", "gke", "vsphere")
-
-				By("creating a StorageClass")
-				class := newStorageClass("", "internal")
-				class, err := c.StorageV1beta1().StorageClasses().Create(class)
-				defer c.StorageV1beta1().StorageClasses().Delete(class.Name, nil)
-				Expect(err).NotTo(HaveOccurred())
-
-				By("creating a claim with a dynamic provisioning annotation")
-				claim := newClaim(ns)
-				claim.Spec.StorageClassName = &class.Name
-
-				defer func() {
-					c.Core().PersistentVolumeClaims(ns).Delete(claim.Name, nil)
-				}()
-				claim, err = c.Core().PersistentVolumeClaims(ns).Create(claim)
-				Expect(err).NotTo(HaveOccurred())
-
-				if framework.ProviderIs("vsphere") {
-					// vsphere provider does not allocate volumes in 1GiB chunks, so setting expected size
-					// equal to requestedSize
-					testDynamicProvisioning(c, claim, requestedSize)
-				} else {
-					// Expected size of the volume is 2GiB, because the other three supported cloud
-					// providers allocate volumes in 1GiB chunks.
-					testDynamicProvisioning(c, claim, "2Gi")
-				}
-			})
+	framework.KubeDescribe("DynamicProvisioner", func() {
+		It("should create and delete persistent volumes [Slow] [Volume]", func() {
+			framework.SkipUnlessProviderIs("openstack", "gce", "aws", "gke")
```

Comment:
@divyenpatel You should add vsphere back to all these tests once your CI environment is updated.